ui/job-view/headerbars/TiersMenu.jsx

```diff
@@ -10,15 +10,15 @@ export default function TiersMenu(props) {
 
   return (
     <span className="dropdown">
```

Comment:
I just noticed that this span is using the "dropdown" class, which to me seems like it's using old bootstrap CSS (meaning it's also using some jquery under the hood for interactivity). Could find out if this is the case, and if so, we'll need to switch it to a reactstrap dropdown menu instead.

Reply:
It does use a Bootstrap CSS class, but it may be the one that Reactstrap uses (not sure). Anyway, I have converted it to a Reactstrap Component.

Reply:
Have you tested it to be sure it's replacing it with the same thing? Let's verify before this is merged :) You can see what classes a reactstrap component uses by inspecting it in the browser.

Reply:
@sarah-clements I have re-inserted the id, and some missing classes (checkbox-dropdown-menu) on the `DropdownMenu`.

There are a couple of additional classes, `btn btn-secondary`, that are added automatically in the `DropdownToggle`. Do you think that would be a problem?

Reply:
So this is up to you to determine when making the changes locally - you want to be sure to test the UI  :) what does the `checkbox-dropdown-menu` class do and is it necessary? `btn-secondary` gives the button a dark grey color so you'll need to determine visually if that is ok. If you're not certain, you can post a screenshot and ask.

Reply:
Yes, you are right about `checkbox-dropdown-menu`. It doesn't change the UI.

About the `btn-seconday`, there is only a single difference, when the button is active:
![A screenshot for both master and this branch view for Tiers Menu dropdown. It shows the style difference for the active state of the button](https://user-images.githubusercontent.com/3901809/70931226-b3bafe80-2015-11ea-8e35-b25edec46fd5.jpg)

The active area goes a little beyond the parent div, and it also happens with the Health button (on both master and this branch). I have tested and it can be "fixed" with a CSS change. Can I change in this following commit?

Reply:
Yes, that's fine :)